README.md

```diff
@@ -137,14 +136,21 @@ More information in the [highs-sys crate](https://crates.io/crates/highs-sys).
 
 [highs]: https://highs.dev
 
+### Windows Requirements
+
+Install both CMake and Clang prior to attempting to build or use this project.
+
+- [CMake](https://cmake.org/download/)
+- [Clang](https://community.chocolatey.org/packages/llvm)
+
```

Comment:
These requirements are not specific to windows. You need cmake and a C++ compiler on any platform to compile HiGHS.
 The C++ compiler does not have to be clang. It could also be GCC, or a proprietary compiler.